sirepo/job_supervisor.py

```diff
@@ -493,6 +485,16 @@ def _create_op(self, opName, req, **kwargs):
         self.ops.append(o)
         return o
 
+    def _get_max_run_secs(self, op_name, kind, run_mode):
+        pkdp('{} {} {}', op_name, kind, run_mode)
```

Comment:
pkdlog

Reply:
This should've been on my #1435 branch. Reverted.